Model/Cron.php

```diff
@@ -2068,4 +2169,32 @@ private function addNotificationErrorComment($errorMessage)
             $this->_order->save();
         }
     }
+
+    /**
+     * @param $expirationDate
+     * @return string
+     * @throws \Exception
+     */
+    private function getExpirationDate($expirationDate)
+    {
+        $expirationDate = explode('/', $expirationDate);
+
+        //add leading zero to month
+        $month = sprintf('%02d', $expirationDate[0]);
+
+        $expDate = new \DateTime(
+            $expirationDate[1]
+            . '-'
+            . $month
+            . '-'
+            . '01'
+            . ' '
+            . '00:00:00',
+            new \DateTimeZone('UTC')
+        );
+
+        // add one month
+        $expDate->add(new \DateInterval('P1M'));
+        return $expDate->format('Y-m-d 00:00:00');
```

Comment:
Does the 00:00:00 works with format?
```suggestion
        return $expDate->format('Y-m-d H:i:s');
```